Lesson1/T_Ilya/CPP_ADV_Lesson1/task1.cpp

```diff
@@ -0,0 +1,26 @@
+#include "task.h"
+#include "functions.h"
+
+namespace MySpace{
+int task1() //  -----VECTOR-----
+{
+    std::vector<int> vect_1{};
+    std::vector<int> vect_2{};
+    for(auto i{0}; i < 10; i++){
+        vect_1.push_back(i);
+        vect_2.push_back(i);
+    }
+    std::cout << "Vector 1:" << std::endl;
+    MFun::print_vect(vect_1);
+    std::cout << "Vector 2:" << std::endl;
+    MFun::print_vect(vect_2);
+    std::vector<int> vect_result{};
+    vect_result.insert(vect_result.end(), vect_1.begin(), vect_1.end());
```

Comment:
You can fill `vect_result` via constructor.

Reply:
????